drivers/spi/spi_renesas_rz_rspi.c

```diff
@@ -164,11 +164,6 @@ static int spi_rz_rspi_configure(const struct device *dev, const struct spi_conf
 	spi_bit_width_t spi_width;
 	fsp_err_t err;
 
-	if (spi_context_configured(&data->ctx, spi_cfg)) {
```

Comment:
No, the bug is in the caller: spi_config are meant to be constant. This to avoid re-configuring unnecessarily the spi-controller each time your want to r/w through it.